/**:
  ros__parameters:
    # If the number of consecutive invalid trajectory exceeds this threshold, the Diag will be set to ERROR.
    # (For example, threshold = 1 means, even if the trajectory is invalid, Diag will not be ERROR if
    #  the next trajectory is valid.)
    diag_error_count_threshold: 0

    display_on_terminal: false # show error msg on terminal

    thresholds:
      max_distance_deviation: 1.0
      rolling_back_velocity: 0.5
      over_velocity_offset: 2.0
      over_velocity_ratio: 0.2

    vel_lpf_gain: 0.9 # Time constant 0.33
    hold_velocity_error_until_stop: true
